packages/graphql-server-module-graphiql/src/renderGraphiQL.ts

```diff
@@ -70,7 +70,7 @@ export function renderGraphiQL(data: GraphiQLData): string {
   <script src="//cdn.jsdelivr.net/react/15.0.0/react-dom.min.js"></script>
   <script src="//cdn.jsdelivr.net/graphiql/${GRAPHIQL_VERSION}/graphiql.min.js"></script>
   ${usingSubscriptions ?
-    '<script src="//unpkg.com/subscriptions-transport-ws@0.5.4/browser/client.js"></script>' +
+    '<script src="//unpkg.com/subscriptions-transport-ws@0.7.0/browser/client.js"></script>' +
```

Comment:
Lets put under const like graphiql version. Also, graphiql version can be updated to 0.10.2